jwe/jwe.go

```diff
@@ -0,0 +1,69 @@
+package jwe
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+)
+
+func NewJWE(alg KeyAlgorithm, key interface{}, method EncryptionType, plaintext []byte) (*jwe, error) {
```

Comment:
Plaintext is the JWT in compact form, correct?

Reply:
Any sensitive info. It can be JWT in compact form.